src/model/quotes.js

```diff
@@ -279,20 +274,31 @@ class QuotesModel {
       let res
       try {
         res = await fetch(fullUrl, opts)
-      } catch (e) {
-        throw new Errors.FSPIOPError('Network error', `Network error forwarding quote request: ${e.stack || util.inspect(e)}`,
-          fspiopSource, Errors.ApiErrorCodes.DESTINATION_COMMUNICATION_ERROR)
+      } catch (err) {
+        throw ErrorHandler.CreateFSPIOPErrorFromErrorCode(1001, `Network error forwarding quote request to ${fspiopDest}`, err, fspiopSource, [
+          { key: 'url', value: fullUrl },
+          { key: 'sourceFsp', value: fspiopSource },
+          { key: 'destinationFsp', value: fspiopDest },
+          { key: 'method', value: opts.method },
+          { key: 'request', value: JSON.stringify(opts) }
+        ])
       }
       this.writeLog(`forwarding quote request ${quoteId} from ${fspiopSource} to ${fspiopDest} got response ${res.status} ${res.statusText}`)
 
       // handle non network related errors below
       if (!res.ok) {
-        throw new Errors.FSPIOPError(res.statusText, 'Got non-success response forwarding quote request',
-          fspiopSource, Errors.ApiErrorCodes.DESTINATION_COMMUNICATION_ERROR)
+        throw ErrorHandler.CreateFSPIOPErrorFromErrorCode(1001, 'Got non-success response forwarding quote request', null, fspiopSource, [
```

Comment:
Please use the enum for errorCode from the Error library

Reply:
Done